db/migrate/20170524011352_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, foreign_key: true
+      t.boolean :like
```

Comment:
this depends on if you want to keep track of likes/dislikes. i don't think you actually need a value here to be honest. you'd either have a record in this table if a user liked something or you wouldn't